src/frontends/pytorch/src/op/grid_sampler.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/grid_sample.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::opset10;
```

Comment:
after you remove `urils.hpp` this will no longer work and you will need to add `ov::op::v*` for each operation

Reply:
removed